6.x/base-how-to.md

```diff
@@ -411,31 +411,38 @@ Add whatever validation rules & inputs you want, in addition to name and passwor
     public function showRegistrationForm()
     {
 
-        return backpack_view('auth.register');
+        // if registration is closed, deny access
+        if (! config('backpack.base.registration_open')) {
+            abort(403, trans('backpack::base.registration_closed'));
+        }
+
+        $this->data['title'] = trans('backpack::base.register'); // set the page title
+
+        return view(backpack_view('auth.register'), $this->data);
     }
 ```
-This will make the registration process pick up a view you can create, in ```resources/views/vendor/backpack/ui/auth/register.blade.php```. You can copy-paste the original view, and modify as you please. Including adding your own custom inputs.
+This will make the registration process pick up a view you can create, in ```resources/views/vendor/backpack/{theme}/auth/register.blade.php```. You can copy-paste the original view, and modify as you please. Including adding your own custom inputs.
```

Comment:
```suggestion
This will make the registration process pick up a view you can create, in ```resources/views/vendor/backpack/{theme}/auth/register.blade.php```. You can copy-paste the original view, and modify as you please. Including adding your own custom inputs. (replace {theme} with the theme you are using, by default is `theme-tabler`)
```